src/Microsoft.Identity.Client/MigrationAid/TokenCache.MigrationAid.cs

```diff
@@ -78,38 +78,38 @@ public void DeserializeUnifiedAndAdalCache(CacheData cacheData)
         }
 
         /// <summary>
-        /// Serializes using the <see cref="SerializeMsalV2"/> serializer.
+        /// Serializes using the <see cref="ITokenCacheSerializer.SerializeMsalV2"/> serializer.
         /// Obsolete: Please use specialized Serialization methods.
-        /// <see cref="SerializeMsalV2"/> replaces <see cref="Serialize"/>.
-        /// <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/> Is our recommended way of serializing/deserializing.
-        /// <see cref="SerializeAdalV3"/> For interoperability with ADAL.NET v3.
+        /// <see cref="ITokenCacheSerializer.SerializeMsalV2"/> replaces <see cref="Serialize"/>.
+        /// <see cref="ITokenCacheSerializer.SerializeMsalV3"/>/<see cref="ITokenCacheSerializer.DeserializeMsalV3"/> Is our recommended way of serializing/deserializing.
+        /// <see cref="ITokenCacheSerializer.SerializeAdalV3"/> For interoperability with ADAL.NET v3.
         /// </summary>
-        /// <returns>array of bytes, <see cref="SerializeMsalV2"/></returns>
+        /// <returns>array of bytes, <see cref="ITokenCacheSerializer.SerializeMsalV2"/></returns>
         /// <remarks>
-        /// <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/> is compatible with other MSAL libraries such as MSAL for Python and MSAL for Java.
+        /// <see cref="ITokenCacheSerializer.SerializeMsalV3"/>/<see cref="ITokenCacheSerializer.DeserializeMsalV3"/> is compatible with other MSAL libraries such as MSAL for Python and MSAL for Java.
         /// </remarks>
         [Obsolete("This is expected to be removed in MSAL.NET v4 and ADAL.NET v6. We recommend using SerializeMsalV3/DeserializeMsalV3. Read more: https://aka.ms/msal-net-3x-cache-breaking-change", false)]
```

Comment:
this is removed (as we now are in MSAL.NET v4)?
Now a link to https://aka.ms/msal-net-4x-cache-breaking-change?